streams/src/test/java/org/apache/kafka/streams/TopologyTest.java

```diff
@@ -1164,15 +1172,19 @@ public void kGroupedStreamZeroArgCountShouldPreserveTopologyStructure() {
                 "      <-- KSTREAM-SOURCE-0000000000\n\n",
             describe.toString()
         );
+
+        assertThat(topology.internalTopologyBuilder.setApplicationId("test").buildTopology().hasPersistentLocalStore(), is(true));
```

Comment:
I agree! But since today is the end of v3.2.0 feature freeze date, I created [KAFKA-13747](https://issues.apache.org/jira/browse/KAFKA-13747) for this test refactoring. Will work on it later. Thank you.